xbmc/pvr/channels/PVRChannelGroup.cpp

```diff
@@ -601,7 +610,7 @@ bool CPVRChannelGroup::RemoveFromGroup(const CPVRChannelPtr &channel)
   return bReturn;
 }
 
-bool CPVRChannelGroup::AddToGroup(const CPVRChannelPtr &channel, const CPVRChannelNumber &channelNumber, bool bUseBackendChannelNumbers)
+bool CPVRChannelGroup::AddToGroup(const CPVRChannelPtr &channel, const CPVRChannelNumber &channelNumber, int iOrder, bool bUseBackendChannelNumbers)
```

Comment:
Do you think you could left align all the & here and in every other affected line you touched to make these lines obey our coding guidelines?

Reply:
Done